test/e2e/specs/site-editor/pages.spec.js

```diff
@@ -141,13 +141,16 @@ test.describe( 'Pages', () => {
 			.getByRole( 'button', { name: 'Move down', exact: true } )
 			.click();
 
+		await editor.canvas
+			.getByRole( 'textbox', { name: 'Site title text' } )
+			.click( { force: true } );
 		await editor.canvas
 			.getByRole( 'textbox', { name: 'Site title text' } )
 			.fill( 'New Site Title' );
```

Comment:
@kevin940726 this is a strange one - the test broke on the fill title but I couldn't see an obvious reason why. The main change is we are now navigating to a different entity in focused mode, but the above line to move the block works, so new entity definitely loaded. 

The only way I could get site title text to update was this forced click or running `.fill` twice and it worked the second time. Any thoughts on why this one call make have broken with these changes, and if there is a better way than the forced click to fix it?